client/styles/components/_nav.scss

```diff
@@ -103,6 +103,29 @@
   }
 }
 
+.nav__items-right--logged-out {
```

Comment:
i think it would make sense to treat the nav changes in a separate PR—changing the panels to be vertical instead of horizontal feels like a no-brainer, whereas the nav changes on mobile i think need a larger re-design.

Reply:
Makes sense to me. I can revert these.